go-controller/pkg/ovn/management-port.go

```diff
@@ -19,7 +19,7 @@ const (
 func configureManagementPortWindows(clusterSubnet, clusterServicesSubnet,
 	routerIP, interfaceName, interfaceIP string) error {
 	// Up the interface.
-	args := []string{"Enable-NetAdapter", fmt.Sprintf("%s", interfaceName)}
+	args := []string{"Enable-NetAdapter", "-IncludeHidden", fmt.Sprintf("%s", interfaceName)}
```

Comment:
Can you put the changes of the go files in a separate commit?

Reply:
Ofc

Reply:
https://github.com/openvswitch/ovn-kubernetes/pull/372
sent